### DISCUSSION TITLE
How to set boat motion and float at the same time

### DISCUSSION THREAD

**Discussion by hongshanma** [DualSPHysics/DualSPHysics#260] [DUALSPHYSICS_V5.2]
Help!!! anyone can help me!!!!  dualpshysics can not set motion and float at the same time.But i really need boat move and float

**Comment by wangjiahan276** [comment]
DualSPHysics_v5.2.2\DualSPHysics_v5.2\examples\others\MotionFloating,this example may give you some inspiration

**Reply by hongshanma** [reply]
Thank U.     I have sloved this problem
